sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -241,7 +246,11 @@ public abstract static class Read extends PTransform<PBegin, PCollection<Row>> {
 
     static Read create() {
       BigtableConfig config =
-          BigtableConfig.builder().setTableId(StaticValueProvider.of("")).setValidate(true).build();
+          BigtableConfig.builder()
+              .setTableId(StaticValueProvider.of(""))
+              .setValidate(true)
+              .setDataflowThrottleReporting(false)
```

Comment:
I think you don't need to set it here again since you're already setting it in BigtableConfig line 80